src/main/java/com/plcok/archive/controller/ArchiveController.java

```diff
@@ -127,4 +127,24 @@ public ResponseEntity<Void> deleteArchive(@AuthenticationPrincipal User user,
         service.deleteArchive(user, archive);
         return ResponseEntity.noContent().build();
     }
+
+    @Operation(summary = "아카이브 좋아요")
+    @ApiResponse(responseCode = "204")
+    @PostMapping("/archives/{archiveId}/like")
+    public ResponseEntity<Void> like(@AuthenticationPrincipal User user,
+                                     @Entity(name = "archiveId") Archive archive,
+                                     @PathVariable(name = "archiveId") long ignoredArchiveId) {
+        service.like(user, archive);
+        return ResponseEntity.noContent().build();
+    }
+
+    @Operation(summary = "아카이브 좋아요 해제")
+    @ApiResponse(responseCode = "204")
+    @DeleteMapping("/archives/{archiveId}/dislike")
```

Comment:
```suggestion
    @DeleteMapping("/archives/{archiveId}/like/{isLike}")
```

https://github.com/PLCOK-2024/server/blob/18dcc078dc9fb06ae7320b0622c24aee9da7f2bd/src/main/java/com/plcok/archive/controller/ArchiveController.java#L111-L119
여기랑 일관되게 작업하는게 좋아보여요

Reply:
공개 비공개는 isPublic이라는 boolean 값으로 컨트롤하는데 좋아요는 팔로우처럼 레코드를 생성하고 삭제해야해서 지금 방법이 더 적합한것 같은데 어떻게 생각하세요??

Reply:
```java
if (isLike) {
   service.like(user, archive);
} else {
    service.dislike(user, archive);
}
```

최대한 수정 없이 요청만 일관되게 맞출꺼 생각했어요.

레코드로 관리해도 크게 다르지 않을것 같아요.

Reply:
```java
if (isLike) {
   service.like(user, archive);
} else {
    service.dislike(user, archive);
}
```

최대한 수정 없이 요청만 일관되게 맞출꺼 생각했어요.

레코드로 관리해도 크게 다르지 않을것 같아요.

Reply:
아하 넵넵 알겠습니다. 수정하겠습니다. 감사합니다.